portafly/README.md

```diff
@@ -1,6 +1,31 @@
 # PortaFly
 A React SPA crafted with the ambitious purpose of replacing Porta UI, currently residing inside the Rails App.
 
+## Getting up and running
+1. Clone the `porta` repository to your local machine
+2. Create a `.env` file:
+  * Create a new file in the `/portafly` root folder, and name it `.env`
+  * Paste `REACT_APP_API_HOST="https://multitenant-admin.preview01.3scale.net"` into it, and Save
```

Comment:
This is only needed of you are creating your access token in the preview environment, you could create it in your local environment

Reply:
true, maybe a way to describe the necessity of having an instance running of Porta project. Also, our preview env shouldn't be public.

Reply:
so we suggest running Porta locally first in order to be able to generate the access token, correct?

Reply:
yeah I prefer we use it locally only